pkg/config/generate.go

```diff
@@ -388,3 +388,18 @@ func Generate(cs *api.OpenShiftManagedCluster, pluginConfig api.PluginConfig) (e
 
 	return
 }
+
+func regenerate(cert certificate) bool {
```

Comment:
call this needsGenerate and reverse the sense - ie where it's returning false currently return true, and vv